src/docfx/build/xref/XrefMap.cs

```diff
@@ -48,139 +48,135 @@ public static async Task<XrefMap> Create(Context context, Docset docset)
-                return result;
+                result.Add(uid, conflict.First());
             }
+            return result;
+        }
 
-            private static XrefSpec LoadMarkdown(string content, Document file)
+        private static XrefSpec LoadMarkdown(string content, Document file)
+        {
+            var (_, markup) = Markup.ToHtml(content, file, null, null, null, null, MarkdownPipelineType.ConceptualMarkdown);
+            var uid = markup.Metadata.Value<string>("uid");
+            var title = markup.Metadata.Value<string>("title");
+            if (!string.IsNullOrEmpty(uid))
             {
-                var (_, markup) = Markup.ToHtml(content, file, null, null, null, null, MarkdownPipelineType.ConceptualMarkdown);
-
-                var uid = markup.Metadata.Value<string>("uid");
-                var title = markup.Metadata.Value<string>("title");
-                if (!string.IsNullOrEmpty(uid))
+                var xref = new XrefSpec
                 {
-                    return new XrefSpec
-                    {
-                        Uid = uid,
-                        Name = string.IsNullOrEmpty(title) ? uid : title,
-                        Href = file.SitePath,
-                    };
-                }
-                return null;
+                    Uid = uid,
+                    Href = file.SitePath,
+                };
+                xref.ExtensionData["name"] = string.IsNullOrEmpty(title) ? uid : title;
+                return xref;
             }
+            return null;
+        }
+
+        private static XrefSpec LoadSchemaDocument(List<Error> errors, JToken token, Document file)
+        {
+            var extensionData = new JObject();
 
-            private static XrefSpec LoadSchemaDocument(List<Error> errors, JToken token, Document file)
+            // TODO: for backward compatibility, when #YamlMime:YamlDocument, documentType is used to determine schema.
+            //       when everything is moved to SDP, we can refactor the mime check to Document.TryCreate
+            var obj = token as JObject;
+            var schema = file.Schema ?? Schema.GetSchema(obj?.Value<string>("documentType"));
+            if (schema == null)
             {
-                XrefSpec xref = null;
+                throw Errors.SchemaNotFound(file.Mime).ToException();
+            }
 
-                // TODO: for backward compatibility, when #YamlMime:YamlDocument, documentType is used to determine schema.
-                //       when everything is moved to SDP, we can refactor the mime check to Document.TryCreate
-                var obj = token as JObject;
-                var schema = file.Schema ?? Schema.GetSchema(obj?.Value<string>("documentType"));
-                if (schema == null)
+            var (schemaErrors, content) = JsonUtility.ToObject(token, schema.Type, transform: TransformContent);
+            errors.AddRange(schemaErrors);
+            var uid = obj.Value<string>("uid");
```

Comment:
>var uid = obj.Value<string>("uid"); [](start = 12, length = 35)

Need to support multiple UIDS if more reference pages migrated to SDP. Some examples: [https://docs.microsoft.com/en-us/dotnet/api/system.string.endswith?view=netframework-4.7.2](https://docs.microsoft.com/en-us/dotnet/api/system.string.endswith?view=netframework-4.7.2) [https://docs.microsoft.com/en-us/javascript/api/@azure/cosmos/?view=azure-node-latest](https://docs.microsoft.com/en-us/javascript/api/@azure/cosmos/?view=azure-node-latest)

Reply:
https://github.com/dotnet/docfx/issues/3457 created a new issue